docs/console/company-configuration/events.mdx

```diff
@@ -461,6 +458,91 @@ hence there is no guarantee that the service will be found in the Project config
 </Tabs>
 
 
+### Configuration saved
+
+This event will be triggered when the Project configuration is [successfully saved from Console Design section](/development_suite/api-console/api-design/overview.md#how-to-save-your-configuration).
+
+<Tabs>
+<TabItem value="configuration-saved-table" label="Event Payload" default>
+
+|       Property       |  Value   |                              Description                               |
+| -------------------- | -------- | ---------------------------------------------------------------------- |
+| `tenantId`           | `string` | The identifier of the Company the service belongs to                   |
+| `projectId`          | `string` | The identifier of the Project the service belongs to                   |
+| `revisionName`       | `string` | The name of the saved revision (branch when using the standard workflow) |
+| `savedChangesId`     | `string` | The changes id (commit for project using the standard workflow) used to register the saved configuration |
+| `previousChangesId`  | `string` | The changes id (commit for project using the standard workflow) the current save started from |
+| `startRef`           | `object` | Provided when saving on a new revision/branch                          |
+| `startRef.name`      | `string` | Name of the original revision/branch/version/tag                       |
+| `startRef.type`      | `string` | Type of the original ref used to start the current save, can be revision/version or branch/tag based on the used workflow |
+| `author`             | `object` | Contains information about the save author                             |
+| `author.id`          | `string` | Contains the id of the user performing the save                        |
+| `author.name`        | `string` | Contains the name of the user performing the save                      |
```

Comment:
Is this influenced by the `"x-confidentiality": true` property? Will the name be shown as redacter or something like that? If so, would it be better to add some details here?

Reply:
`x-confidential` only prevents the field to be saved in the webhook invocations history